lib/PHPCfg/Op/Expr/Param.php

```diff
@@ -51,7 +53,7 @@ public function __construct(
         $this->declaredType = $type;
         $this->byRef = $byRef;
         $this->variadic = $variadic;
-        $this->attrGroups = $attrGroups;
+        $this->setAttributeGroups(...$attrGroups);
```

Comment:
Why using the `...` operator?  `$attrGroups` is defined as an `array` on the `__construct` param.